### PR TITLE
DCS-1489 adding sex and CRO number

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/PrisonerSearchServiceTest.kt [hunk 1]
```diff
@@ -172,13 +201,13 @@ class PrisonerSearchServiceTest {
...
     whenever(client.getPrisoner(any())).thenReturn(
-      MatchPrisonerResponse(FIRST_NAME, LAST_NAME, DOB, PRISON_NUMBER, PNC_NUMBER, CRO_NUMBER, INACTIVE_OUT)
+      MatchPrisonerResponse(FIRST_NAME, LAST_NAME, DOB, PRISON_NUMBER, PNC_NUMBER, CRO_NUMBER, GENDER, INACTIVE_OUT)
     )
 
     val prisoner = service.getPrisoner("A1234AA")
 
     assertThat(prisoner).isEqualTo(
-      PrisonerDetails(FIRST_NAME, LAST_NAME, DOB, PRISON_NUMBER, PNC_NUMBER, CRO_NUMBER, false)
+      PrisonerDetails(FIRST_NAME_FORMATTED, LAST_NAME_FORMATTED, DOB, PRISON_NUMBER, PNC_NUMBER, CRO_NUMBER, false)
```
Comment (louisenorris) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/PrisonerSearchServiceTest.kt:210]: Should GENDER be included here too?

Comment (michalnawrockidj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/PrisonerSearchServiceTest.kt]: sorted

**Review by louisenorris — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/Arrival.kt [hunk 2]
```diff
@@ -61,7 +61,14 @@ data class PotentialMatch(
...
 
   @Schema(description = "PNC number", example = "01/1234X")
-  val pncNumber: String?
+  val pncNumber: String?,
+
+  @Schema(description = "CRO number", example = "29906/12J")
+  val croNumber: String?,
+
+  @Schema(description = "Sex", example = "Female")
+  val sex: String?
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/Arrival.kt:70]: sex isn't optional when being retrieved from prison details

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonerDetails.kt [hunk 3]
```diff
@@ -26,5 +26,8 @@ data class PrisonerDetails(
   @Schema(description = "CRO number", example = "SF80/655108T")
   val croNumber: String?,
 
+  @Schema(description = "Sex", example = "Female")
+  val sex: String?,
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonerDetails.kt:30]: and here

**Pull request merged**